app/graphql/types/category.rb

```diff
@@ -0,0 +1,46 @@
+class Types::Category < Types::BaseObject
+  description 'Information about a specific Category'
+
+  field :id, ID, null: false
+
+  field :title, String,
+    null: false,
+    description: 'The name of the category.'
+
+  field :description, String,
+    null: true,
+    description: 'A description of the category.'
+
+  field :slug, String,
+    null: false,
+    description: 'The URL-friendly identifier of this Category.'
+
+  field :anidb_id, ID,
+    null: true,
+    description: 'Kitsu category mapping to Anidb category.'
+
+  field :total_media_count, Integer,
```

Comment:
I think it would be better if we have a media connection on this and expose the count in there. For now, maybe leave it off?